mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -457,20 +457,20 @@ public void modSeqLessThanShouldReturnUidsOfMessageHavingAGreaterModSeq() throws
 
     @Test
     public void sizeGreaterThanShouldReturnUidsOfMessageExceedingTheSpecifiedSize() throws Exception {
-        // Only message 7 is over 10 KB
```

Comment:
The seventh mail was the html one, which i rewrite to english instead of french.
Then, it's smaller than the original one.